docs/userdoc/ContainerInterfaceModel.md

```diff
@@ -1,6 +1,7 @@
 # Container Interface Model
 
-The behavior described in this section is completely hidden from AutoPas users. 
+The behavior described in this section is completely hidden from AutoPas users.
+To ensure functionality, particles must not move more than `verletSkinPerTimestep / 2` per timestep.
```

Comment:
This should be moved under the static rebuilding section as it does not apply to dynamic

Reply:
done, see [4da4355](https://github.com/AutoPas/AutoPas/pull/821/commits/4da43550d82b376771ada1352a61fc648ff9e0d9)